flink-benchmark/multi_size_sliding_feature_view.py

```diff
@@ -0,0 +1,153 @@
+#  Copyright 2022 The Feathub Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import time
+from datetime import timedelta
+
+from feathub.common import types
+from feathub.common.types import Int64
+from feathub.feathub_client import FeathubClient
+from feathub.feature_tables.sinks.black_hole_sink import BlackHoleSink
+from feathub.feature_tables.sources.datagen_source import DataGenSource, RandomField
+from feathub.feature_views.feature import Feature
+from feathub.feature_views.sliding_feature_view import SlidingFeatureView
+from feathub.feature_views.transforms.sliding_window_transform import (
+    SlidingWindowTransform,
+)
+from feathub.table.schema import Schema
+
+if __name__ == "__main__":
```

Comment:
The PR seems to assume that each Flink benchmark will take the form of a python file under the directory `flink-benchmark`, and all those tests share the same docker environment.

This assumption might not hold in the long term. For example, we might want to benchmark the speed that Flink processor writes to Redis. So the docker file for this test needs to start a Redis server. But most other tests will not need to start Redis server.

How about we still follow the same directory structure as existing examples for now? For example, this PR can add files in the directory`feathub-examples/flink-sliding-feature-view-benchmark`.